src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/Conversation.java

```diff
@@ -58,11 +58,11 @@ public abstract class Conversation extends DomainObject {
     private User creator;
 
     @JsonIgnoreProperties("conversation")
-    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.LAZY)
+    @OneToMany(mappedBy = "conversation", cascade = CascadeType.REMOVE, fetch = FetchType.EAGER)
```

Comment:
Do we really want to load the conversation participants every time we retrieve a conversation from the database?
If so, please also adapt the usage of `loadConversationWithParticipantsIfGroupChat` and any other occurrences that interact with the conversation participants.

Reply:
Maybe related: we have a few server tests that limit the number of database calls for conversations. These tests are now failing with 18 instead of 10 and 9 instead of 7 database calls.

Reply:
@JohannesStoehr The exception is thrown because of the Getter method loadConversationWithParticipantsIfGroupChat method is not called